### PR TITLE
add NFT endpoints

### PR DESCRIPTION
**Pull request by idanya**

### CODE REVIEW

**Review by salickco — CHANGES_REQUESTED** [review]
lets align token id param name in all functions - (id(s) or token_id(s))

**Review by salickco — APPROVED** [review]

File: fireblocks_sdk/sdk.py [hunk 1]
```diff
@@ -47,6 +47,70 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
...
+
+    def refresh_nft_ownership_by_vault(self, blockchain_descriptor: str, vault_id: str):
+        """
+
+        :param blockchain_descriptor:
+        :param vault_id:
+        :return:
+        """
+        url = "/v1/nfts/ownership/tokens"
+        body = {
```
Comment (or109) [inline fireblocks_sdk/sdk.py:85]: I might sure it's query param and not body

File: fireblocks_sdk/sdk.py [hunk 2]
```diff
@@ -47,6 +47,70 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
         self.token_provider = SdkTokenProvider(private_key, api_key)
         self.timeout = timeout
 
+    def get_nft(self, token_id: str):
+        url = "/v1/nfts/tokens/" + token_id
+
+        return self._get_request(url)
+
+    def get_nfts(self, token_ids: str):
```
Comment (or109) [inline fireblocks_sdk/sdk.py:55]: Let's decide for a string array or string concatenation string first 🙏

File: fireblocks_sdk/sdk.py [hunk 3]
```diff
@@ -47,6 +47,70 @@ def __init__(self, private_key, api_key, api_base_url="https://api.fireblocks.io
...
+            params['blockchainDescriptor'] = blockchain_descriptor
+
+        if vault_account_id:
+            params['vaultAccountId'] = vault_account_id
+
+        if token_ids:
+            params['ids'] = token_ids
+
+        if params:
+            url = url + "?" + urllib.parse.urlencode(params)
```
Comment (or109) [inline fireblocks_sdk/sdk.py:110]: WDYT for having a function that do this?

```
if params:
            url = url + "?" + urllib.parse.urlencode(params)
```

So you can reuse it

**Review by or109 — CHANGES_REQUESTED** [review]
Small comments

File: fireblocks_sdk/sdk.py (hunk 2 above)
Comment (idanya) [reply fireblocks_sdk/sdk.py]: Done

File: fireblocks_sdk/sdk.py (hunk 3 above)
Comment (idanya) [reply fireblocks_sdk/sdk.py]: Moved to GET func

**Pull request merged**